posthog/settings/web.py

```diff
@@ -257,7 +257,7 @@
 LOGOUT_URL = "/logout"
 LOGIN_REDIRECT_URL = "/"
 APPEND_SLASH = False
-CORS_URLS_REGEX = r"^(/site_app/|/api/(?!early_access_features|surveys|web_experiments).*$)"
+CORS_URLS_REGEX = r"^(/site_app/|/array/|/api/(?!early_access_features|surveys|web_experiments).*$)"
```

Comment:
there's a config somewhere in this file iirc to enable gzip for these new endpoints

Reply:
think i found it